### DISCUSSION TITLE
Access previous published project

### DISCUSSION THREAD

**Discussion by AvdHerik** [phcode-dev/phoenix#1199] [Q&A] [answered]
Hi,

I built a small site with phcode-dev Phoenix and published it a couple of months ago. It was and is actually all working fine till now. But I need to make a change to the index.html file, but how to do this?
The recent projects are gone in the library and when I open the file folder from my desktop, that will still work but when I publish it again it will get a new directory. The problem with that is that I create a QR Code that links to the old published directory.

So, is it still possible to change the files of a previously created directory?

Best regards,
Andre

**Comment by abose** [comment, marked as answer]
Hi @AvdHerik 
Looks like the browser was upgdated/cleared the site data as it was not visited in a while. Browsers usually clean site data of unused sites. 
1. Is this google chrome? Which OS?
2. How long since the site was not visited
3. Was browser uninstalled in between? 

The publish feature is only for development purposes and the link is only valid for a period for 30 days as we usually outline that in the dialog: 
![image](https://github.com/phcode-dev/phoenix/assets/5336369/edf4e0c2-06bf-4b8a-bde4-4cd10527e860)

That being said, we dont usually delete the contents so your content should still be retrievable. We do not go in manually see the contents on the published public sites for privacy reasons, but i can help you out to get back the control of the published site .
1. Please give the url of the site to get back.

**Reply by AvdHerik** [reply]
Hi @abose,

Many thanks for your reply and your generosity in helping me out!

I will think about a solution after gaining access again to redirect this mini-site and host it somewhere on one of my own sites. Then the QR-code will still work.

The URL is: https://phcode.site/p/p-6bfdb47eaaa/QR-Social-Sharing-Page/index.html

Again many thanks!

Best regards,
Andre

**Reply by abose** [reply]
Can you drop your email to arun@core.ai , ill have to send over a private key that you have to use to get access.

**Reply by testingusername6789** [reply]
Is there a way to have published links live for more than 30 days? The goal is to have a page live for 1 year+ without manually publishing it each month.

**Reply by abose** [reply]
> Is there a way to have published links live for more than 30 days? The goal is to have a page live for 1 year+ without manually publishing it each month. 

Will add web hoisting and longer publish options for logged in users by july. The thing is, current publish workflow is anonymous and there is a chance of misuse. So we just delete content every 30 days instead of having to deal with TOS Violations. With logins, we should be able to add permanant previews and more sharing features.
